platform/platform-app/src/main/resources/features.xml

```diff
@@ -1050,7 +1050,7 @@
 
     <feature name="platform-all" install="manual" version="${project.version}" hidden="true">
         <feature prerequisite="true">platform-security-session</feature>
-        <feature>platform-migrate-all</feature>
+        <feature prerequisite="true">platform-migrate-all</feature>
```

Comment:
just double checking - but `platform-migrate-all` doesn't need to wait for `platform-security-session`. My guess would be that we would see these bundles getting started at the same time now.